package.json

```diff
@@ -80,7 +80,7 @@
     "@glimmer/component": "1.1.2",
     "@glimmer/tracking": "1.1.2",
     "@release-it-plugins/lerna-changelog": "6.0.0",
-    "bootstrap": "5.3.2",
+    "bootstrap": "^5.3.2",
```

Comment:
Why are you changing the bootstrap dependency to a version range instead of a pinned version?

Reply:
Ah, that must have happened when I tried fixing the `package.json` after a local ember-try run went wrong. I've reverted it.